index.html

```diff
@@ -29,6 +29,9 @@
                 }, {
                     value: 'Feedback / discussion in the WICG',
                     href: 'https://discourse.wicg.io/t/explicit-api-for-visual-viewport/1399'
+                }, {
+                    value: 'Viewport segments discussion',
+                    href: 'https://www.w3.org/2021/02/24-webscreens-irc'
```

Comment:
nit: Maybe this link is a bit easier on the eyes. This is the html version of the minutes.
https://www.w3.org/2021/02/24-webscreens-minutes

(sorry for the drive-by comment, and you can ignore it, if you prefer the text version)